tests/sentry/utils/test_committers.py

```diff
@@ -281,6 +281,177 @@ def test_java_sdk_path_mangling(self):
+                                        "filename": "View.java",
+                                        "abs_path": "View.java",
+                                        "lineno": 835,
+                                        "in_app": False,
+                                    },
+                                    {
+                                        "function": "performClickInternal",
+                                        "module": "android.view.View",
+                                        "filename": "View.java",
+                                        "abs_path": "View.java",
+                                        "lineno": 7432,
+                                        "in_app": False,
+                                    },
+                                    {
+                                        "function": "performClick",
+                                        "module": "com.google.android.material.button.MaterialButton",
+                                        "filename": "MaterialButton.java",
+                                        "abs_path": "MaterialButton.java",
+                                        "lineno": 1119,
+                                        "in_app": False,
+                                    },
+                                    {
+                                        "function": "performClick",
+                                        "module": "android.view.View",
+                                        "filename": "View.java",
+                                        "abs_path": "View.java",
+                                        "lineno": 7455,
+                                        "in_app": False,
+                                    },
+                                    {
+                                        "function": "onClick",
+                                        "module": "com.jetbrains.kmm.androidApp.MainActivity$$ExternalSyntheticLambda0",
+                                        "lineno": 2,
+                                        "in_app": True,
+                                    },
+                                    {
+                                        "function": "$r8$lambda$hGNRcN3pFcj8CSoYZBi9fT_AXd0",
+                                        "module": "com.jetbrains.kmm.androidApp.MainActivity",
+                                        "lineno": 0,
+                                        "in_app": True,
+                                    },
+                                    {
+                                        "function": "onCreate$lambda-1",
+                                        "module": "com.jetbrains.kmm.androidApp.MainActivity",
+                                        "filename": "MainActivity.kt",
+                                        "abs_path": "MainActivity.kt",
+                                        "lineno": 55,
+                                        "in_app": True,
```

Comment:
Core filename munging is the same for basic java and kotlin - concat `filename` and `module` to form the fully-qualified file. Instead of .java, its .kt